src/Context/ApiContext.php

```diff
@@ -299,6 +301,60 @@ protected function compareStructureResponse(string $variableFields, PyStringNode
         }
     }
 
+    /**
+     * @param array $requestParams
+     *
+     * @When I have :requestParams with runnable values (placed in <>)
+     *
+     * @Then :requestParams should be converted with executing runnable code
```

Comment:
Is it just to describe the method or does it do something?

Reply:
Just to describe. Already removed it.